.readthedocs.yml

```diff
@@ -10,8 +10,10 @@ sphinx:
 conda:
     environment: docs/environment-rtd.yml
 
-python:
-    version: 3.8
-    install:
-        - method: setuptools
-          path: .
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "mambaforge-22.9"
```

Comment:
I love that this takes the build time down from [1265s](https://readthedocs.org/projects/pygeoprocessing/builds/21832922/) to [276s](https://readthedocs.org/projects/pygeoprocessing/builds/21970683/)